sdk/appconfiguration/Azure.Data.AppConfiguration/samples/README.md

```diff
@@ -0,0 +1,73 @@
+# Azure App Configuration client SDK samples
+
+This directory contains samples for the Azure App Configuration client SDK.  Each sample illustrates one or more capabilities of the library.
+
+These include:
+ - [Hello World](#hello-world): Create and delete a configuration setting.
+ - [Hello World Extended](#hello-world-extended): Asynchronously create, update and delete configuration settings with labels.
+ - [Set Read Only](#set-read-only): Make a configuration setting read only, and then return it to a read write state.
```

Comment:
Nit: similar to above, seeing "read only" and "read write" when I more typically see these as "read-only" or "read-write", which I remember you using in your code samples I reviewed earlier today.